kpi/serializers/v2/asset_counts.py

```diff
@@ -0,0 +1,45 @@
+import datetime
+
+from django.utils import timezone
+from rest_framework import serializers
+
+from kpi.constants import PERM_VIEW_SUBMISSIONS, PERM_PARTIAL_SUBMISSIONS
+from kpi.utils.object_permission import get_database_user
+
+
+class AssetCountsSerializer(serializers.Serializer):
+    daily_submission_counts = serializers.SerializerMethodField()
+    total_submission_count = serializers.SerializerMethodField()
+
+    class Meta:
+        fields = ('daily_submission_counts', 'total_submission_count')
+
+    def get_daily_submission_counts(self, asset):
+        today = timezone.now().date() + datetime.timedelta(days=1)
```

Comment:
`get_daily_counts` already handles it for us. No need to add the extra timedelta.